hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OzoneFSUtils.java

```diff
@@ -75,6 +75,24 @@ public static String getImmediateChild(String descendant, String ancestor) {
     return descendant;
   }
 
+  /**
+   * The function returns file name from the given absolute path. For
+   * example, the given key path '/a/b/c/d/e/file1' then it returns file name
+   * 'file1'.
+   */
+  public static String getDirName(String keyName) {
+    if (keyName.endsWith("/")) {
```

Comment:
Can we use the import java.nio.file.Path here in place of string manipulation and then use path.getname here ?